src/Crud/Controller/Create.php

```diff
@@ -42,8 +43,8 @@ public function __construct(DataStoreInterface $dataStore, FormFactoryInterface
     public function __invoke(Request $request)
     {
         // Metadata you can specify in routing definition
-        $modelClass = $request->attributes->get('melodiia_model');
-        $form = $request->attributes->get('melodiia_form');
+        $modelClass = $request->attributes->get(self::MODEL_ATTRIBUTE);
+        $form = $request->attributes->get(self::FORM_ATTRIBUTE);
```

Comment:
`$formData` ? :D

Reply:
This is the FQCN of the FormType. Nothing related to `formdata` here. Or maybe I misunderstand your comment?